### ISSUE TITLE
"Security Phrase" field is in bold during login

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-web#19006]
### Steps to reproduce

1. Where are you starting? What can you see?

Upon logging in and choosing the option to use my security passphrase to verify my device, I found that the placeholder in the field for entering my security passphrase is oddly bolded.

### What did you expect?

Field placeholders are not usually bold.

### What happened?

![image](https://user-images.githubusercontent.com/1342360/133083662-ead13c1c-21db-4ce8-bec8-15f5415dd9e3.png)

### Operating system

Arch Linux

### Browser information

Firefox v91.0.2

### URL for webapp

develop.element.io

### Have you submitted a rageshake?

No

**Comment by neer17** [comment]
Hi everyone! I have started working on it, hoping that's okay.

**Comment by neer17** [comment]
Tried logging in with different browsers and found out that it is only "Firefox" specific. Working on the issue, would create a PR soon.

**Comment by SimonBrandner** [comment]
> Hi everyone! I have started working on it, hoping that's okay.

Awesome!

**Comment by anoadragon453** [comment]
@neer17 Other than the colour, there might also be the concern that the font weight of the placeholder is too high on firefox (bold vs. regular)?

**Comment by neer17** [comment]
Yeah, that's right. I tried giving `color:#a7a7a7;font-weight:400` to make it look consistent but still, both placeholders are looking different in Firefox and Chrome. However, in Chrome and Edge the text styles appear to be the same. I can only think of fixing this issue in Firefox by "hit and try" the opacity value or the color till it matches Chome's.
